internal/container/deviceservice.go

```diff
@@ -22,6 +22,9 @@ var ProtocolDriverName = di.TypeInstanceToName((*interfaces.ProtocolDriver)(nil)
 // AutoEventManagerName contains the name of autoevent manager implementation in the DIC
 var AutoEventManagerName = di.TypeInstanceToName((*interfaces.AutoEventManager)(nil))
 
+// ProtocolDiscoveryName contains the name of protocol discovery implementation in the DIC.
+var ProtocolDiscoveryName = di.TypeInstanceToName((*interfaces.ProtocolDiscovery)(nil))
```

Comment:
rename the term `ProtocolDiscovery` to `ProfileScan`, including the function and interface name